readme.md

```diff
@@ -332,7 +332,7 @@ cat apps/settings.py | less
 **STOP CMD/CTRL + F IN A FOLDER** :-1:
 
 ```shell
-grep -ir "Query" file.txt
+grep -i "Query" file.txt
```

Comment:
CC @umr55766

Reply:
Thanks for reaching out @ashleyharvey and @stevemao for tagging me.

TBH, I had added `flag i` and `flag r` because I use that almost always with grep command.
And I just checked my original PR and found that it was only an example (& command syntax was separate), you can find those here https://github.com/you-dont-need/You-Dont-Need-GUI/pull/33/files.